livekit-plugins/livekit-plugins-gladia/README.md

```diff
@@ -0,0 +1,76 @@
+# LiveKit Plugins Gladia
```

Comment:
should be called "Gladia plugin for LiveKit".

Thank you so much for writing a good README! we need this on all our plugins.